cool-demos/slash-commands.html

```diff
@@ -0,0 +1,247 @@
+                },
+                {
+                    text: 'Media...',
+                    plugin: 'media',
+                    icon: 'embed',
+                    action: () => editor.execCommand('mceMedia')
+                },
+                {
+                    text: 'Quote',
+                    icon: 'quote',
+                    action: () => editor.execCommand('mceBlockQuote')
+                },
+                {
+                    text: 'Code block...',
+                    icon: 'code-sample',
+                    action: () => editor.execCommand('CodeSample')
+                },
+            ]
+
+            // Set up the autocompleter
+            // https://www.tiny.cloud/docs/ui-components/autocompleter/
+            editor.ui.registry.addAutocompleter('insertActions', {
+
+                // The character to trigger the autocompleter,
+                // the number of characters to type before displaying the autocompleter menu and,
+                // configure the autocompleter to display a menu opposed a row of icons
+                // https://www.tiny.cloud/docs/ui-components/autocompleter/#configurationoptions
+                ch: '/',
+                minChars: 0,
+                columns: 1,
+
+                // The fetch function is called when the trigger char is pressed
+                // and the matches predicate returns true.
+                // In other words, this function (re)builds the menu as the user type.
+                fetch: function (pattern) {
+
+                    const matchedActions = insertActions.filter((action) => {
+                        // Normalize the input to match the same action regardless if
+                        // the end-user used upper or lowercase.
+                        return (
+                            action.type === 'separator' ||
+                            action.text.toLowerCase().indexOf(pattern.toLowerCase()) !== -1
+                        );
+                    }).filter((action, i, actions) => {
+                        // As the end-user filters the list, separators can end up adjacent
+                        // to each other which looks bad. This function resolves that
+                        const prevAction = actions[i - 1];
+                        const nextAction = actions[i + 1];
+                        const isRedundantSeparator = action.type === 'separator' && (
+                            !prevAction ||
+                            !nextAction ||
+                            prevAction.type === 'separator' ||
+                            nextAction.type === 'separator'
+                        );
+                        return !isRedundantSeparator;
+                    });
+
+                    // Here the matched autocompleter objects are built
+                    // https://www.tiny.cloud/docs/ui-components/autocompleter/#autocompleteitem
+                    return new tinymce.util.Promise(function (resolve) {
```

Comment:
This will be removed in v6 since we are dropping IE support will we update this demos to v6 after release? But we need it here now for v5 support if we want these demos to work on IE.